test/unit/services/test-get-overview.js

```diff
@@ -56,4 +64,18 @@ describe('services/get-overview', function () {
       done()
     })
   })
+
+  it('should return a results object with the correct overview details for a team', function (done) {
+    var teamName = 'team'
```

Comment:
I know this is copied from the existing test but can we get these values from `constants/organisation-unit.js` so we're not repeating ourselves.